test_files/objc/primitives.d

```diff
@@ -1,6 +1,10 @@
+import core.stdc.time;
```

Comment:
It is an old bug in imports translation. A matching import is searched with `endsWith`

```
    string isKnownInclude (string include)
    {
        include = Path.stripExtension(include);

        if (auto r = knownIncludes.find!((k, _) => include.endsWith(k)))
            return r.value;

        return null;
    }
```

it creates an ambiguity when there are two or more matches with the same suffix (e.g. `runtime.h` and `time.h`). And as linear search is used first one is used arbitrarily. 

It needs improvement, but I don't want to fix it in this PR.